index.php

```diff
@@ -20,8 +43,41 @@
 </head>
 
 <body>
+    <!-- nav-bar -->
+    <div class='navBar'>
+        <div class='leftNav'>
+            <a class='navLink'>MyRecords</a>
+        </div>
+        <div class='rightNav'>
+            <a class='navLink'>+ Record</a>
+            <a class='navLink'>Archive</a>
```

Comment:
For the time being, can you remove these or even just comment them out? Just because we only ever want to have 'working' code in main, an these features don't yet exist